### PR TITLE
Polyfill requestAnimationFrame for tests (Jest)

### PR DESCRIPTION
**Pull request by bhongy**
This address React warnings when running `yarn test`.

Before:
screen shot 2018-05-07 at 4 21 21 pm

After:
screen shot 2018-05-07 at 4 21 52 pm

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 76e3c753567ab8a13a79e19c1eb413161ed16c72

https://deploy-preview-161--gestalt.netlify.com

File: package.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
     "babel-preset-stage-1": "^6.24.1",
     "concurrently": "^3.5.1",
     "danger": "^3.1.7",
-    "enzyme-adapter-react-16": "^1.0.5",
```
Comment (bhongy) [inline package.json]: Entry got moved by `yarn`. I can revert it.

**Comment by chrislloyd** [comment]
I think this might be fixed by #158 - just checking now.

**Comment by bishwei** [comment]
+1 I don't think this is necessary with jest 22 https://facebook.github.io/jest/blog/2017/12/18/jest-22.html

**Pull request closed without merging**